### ISSUE TITLE
Scroll glitch

### ISSUE THREAD

**Issue by hinst** [RolandPheasant/TailBlazer#150]
Had this glitch today: the bottom line of the document goes up
![1](https://cloud.githubusercontent.com/assets/958706/16190578/b2572c90-36e8-11e6-9ae6-bf49c0906432.png)

**Comment by RolandPheasant** [comment]
Can you repeatedly reproduce that?

**Comment by hinst** [comment]
yes; when more messages are added, it goes up and up. At first there is only one empty line at the bottom; then two then 3 and so on (however the app which writes to log does not write empty lines; and even if it did, then new non-empty lines would appear after empty lines)

**Comment by RolandPheasant** [comment]
That's an old version. 

Could you try again with the latest version.  Here https://github.com/RolandPheasant/TailBlazer/releases

**Comment by hinst** [comment]
I checked just now; I still get this bug with the new version. Looks exactly as on the screenshot I posted with my first message

**Comment by RolandPheasant** [comment]
This one is really odd.  Can you open the tailblazer log file to see whether the problem exists on your machine or is specific to that particular log file

**Comment by hinst** [comment]
I observed the issue today at 17:00. Text went up as high as 50% of the view-panel.
[logger.txt](https://github.com/RolandPheasant/TailBlazer/files/373853/logger.txt)
